bin/cyhy-import

```diff
@@ -79,6 +80,21 @@ def has_intersections(db, nets, filename, owner):
         print_intersections(intersections)
         return True
 
+def has_restricted_ips(nets):
+    reader = geoip2.database.Reader('/usr/local/share/GeoIP/GeoIP2-City.mmdb')
+    bad_countries = ['China','Russia','Iran','North Korea']
+    has_bad_address = False
+
+    for cidr in nets.iter_cidrs():
+        geo_data = reader.city(str(cidr[0]))
+        country_name = geo_data.country.name
+        if country_name in bad_countries:
+            print('{} contains addresses in country that is unavailable due to current scanning policy: {}'.format(cidr,country_name))
+            has_bad_address = True
+    if has_bad_address:
+        return True
+    else:
+        return False
```

Comment:
Updating variable name for consistency and since it's a boolean you can just return it instead of using an if/else.

```suggestion
            has_restricted_address = True

    return has_restricted_address
```